src/SupervisionSuccess/SupervisionSuccessContainer.js

```diff
@@ -49,11 +49,21 @@ const SupervisionSuccessContainer = ({ params, isError }) => {
   const [savings, setSavings] = useState(0);
   const [chartData, setChartData] = useState([]);
 
-  const onStateChange = useCallback((newState) => {
-    setState(newState);
+  const onReset = useCallback(() => {
     setChangeInNewAdmissions(DEFAULT_STATE.changeInNewAdmissions);
     setChangeInRevocations(DEFAULT_STATE.changeInRevocations);
   }, []);
+  const onMaintain2020Levels = useCallback(() => {
+    setChangeInRevocations(-21);
```

Comment:
Let's extract these out to constants named `2020_CHANGE_IN_REVOCATIONS` and `2020_CHANGE_IN_NEW_ADMISSIONS`